skbio/tree/_trie.py

```diff
@@ -177,24 +179,49 @@ class CompressedTrie(object):
 
     """
 
+    @deprecated(as_of="0.4.0", until="0.4.1",
```

Comment:
put the text in a var to avoid duplication (in the future we'll have a better way of deprecating entire classes)